packages/server/package.json

```diff
@@ -12,7 +12,10 @@
     "compile": "tsc -p tsconfig.json",
     "test": "yarn test-db-up && yarn jest && yarn test-db-down",
     "test-db-up": "docker-compose up -d",
-    "test-db-down": "docker-compose down --remove-orphans"
+    "test-db-down": "docker-compose down --remove-orphans",
+    "migrate:create": "cd ./src/application/migrations/ && migrate create --template-file ./migrations/_template.ts",
+    "migrate:up": "cd ./src/application/migrations/ && migrate up --compiler=\"ts:./compiler.js\"",
+    "migrate:down": "cd ./src/application/migrations/ && migrate down --compiler=\"ts:./compiler.js\""
```

Comment:
I don't see migrations being applied anywhere before the server start. Would it make sense to run migrations immediately after  `docker-compose up -d`?

Reply:
Good point. I'll add it.

Reply:
Can't do it like that because we create database dynamically for each test suite. Most likely we'll need a db snapshot.

Reply:
How about renaming migration log so that each file points to different database?

Reply:
Didn't get this one. What do you mean by migration log?

Reply:
Migration log is the file that migration engine uses to track state of applied migrations. You could have a different one per test suite

Reply:
Got it. This is done automatically.